includes/REST/StoreController.php

```diff
@@ -249,7 +249,7 @@ public function get_stores( $request ) {
             $args['meta_query'] = [
```

Comment:
do we really need this meta query if the search item is empty ? @kapilpaul @xaviranik

Reply:
If no stores found in then we are searching again with meta value for store name. Here, we need to remove search and search_columns, because with this args meta_query is not working.

Reply:
what I meant was, if user didn't search for anything, do we need this bit of code? 
`'value'   => isset( $params['search'] ) ? sanitize_text_field( $params['search'] ) : '',`
if search param is empty we can skip I guess?